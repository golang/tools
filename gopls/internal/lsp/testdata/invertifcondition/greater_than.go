package invertifcondition

import (
	"fmt"
	"os"
)

func GreaterThan() {
	if len(os.Args) > 2 { //@suggestedfix("i", "refactor.rewrite", "")
		fmt.Println("A")
	} else {
		fmt.Println("B")
	}
}
