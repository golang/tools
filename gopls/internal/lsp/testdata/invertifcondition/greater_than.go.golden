-- suggestedfix_greater_than_9_2 --
package invertifcondition

import (
	"fmt"
	"os"
)

func GreaterThan() {
	if len(os.Args) <= 2 {
		fmt.Println("B")
	} else { //@suggestedfix("i", "refactor.rewrite", "")
		fmt.Println("A")
	}
}

